permission_handler_apple/ios/Classes/PermissionHandlerEnums.h

```diff
@@ -124,6 +124,7 @@ typedef NS_ENUM(int, PermissionGroup) {
     PermissionGroupPhotosAddOnly,
     PermissionGroupReminders,
     PermissionGroupSensors,
+    PermissionGroupSensorsAlways,
```

Comment:
Same here, revert the reordering and add the `PermissionGroupSensorsAlways ` definition at the bottom of the list.